www/js/transforms/addImageOverflowContainers.js

```diff
@@ -37,8 +32,6 @@ transformer.register( "addImageOverflowXContainers", function( content ) {
     // side to side if needed without causing the entire section to scroll side to side.
     var images = content.getElementsByTagName('img');
     for (var i = 0; i < images.length; ++i) {
-        // Load event used so images w/o style or inline width/height
```

Comment:
This needed to happen *before* image widening because image widening checks for hasOverflowXContainer, but image widening was updated to no longer wait until image 'load'. Because we were still waiting for image 'load' here it no longer happened before image widening.